### PR TITLE
[PublicCloud] Skip validate repo on SLEM 6+

### PR DESCRIPTION
**Pull request by mimi1vx**
https://progress.opensuse.org/issues/163397

Add support for Products Increments in PC tests

VR: 1) http://quasar.suse.cz/tests/179
       2) http://quasar.suse.cz/tests/180

### CODE REVIEW

File: lib/publiccloud/utils.pm [hunk 1]
```diff
@@ -146,6 +146,10 @@ sub register_openstack {
 # Validation for update repos
 sub validate_repo {
     my ($maintrepo) = @_;
+    if (is_sle_micro('>=6.0')) {
+        record_info("NEW MAINT MODE", "New style update, so can't validate repository");
```
Comment (grisu48) [inline lib/publiccloud/utils.pm:150]: Do we really want one info box per repository?

**Review by grisu48 — APPROVED** [review]
LGTM as per https://suse.slack.com/archives/C02CLB8TZP1/p1726743970046109?thread_ts=1726741239.519879&cid=C02CLB8TZP1 but one minor nitpick

File: lib/publiccloud/utils.pm (hunk 1 above)
Comment (jlausuch) [inline lib/publiccloud/utils.pm:150]: The message sounds a bit confusing for whom does not know about this. Maybe something like "Skip repo validation. Embargoed updates are not pushed in SL Micro 6+." or something similar.

Comment (mimi1vx) [reply lib/publiccloud/utils.pm:150]: on SL Micro 6 its only one repo anyway

File: tests/publiccloud/download_repos.pm [hunk 2]
```diff
@@ -74,6 +77,8 @@ sub run {
             script_run("echo 'Downloading $maintrepo ...' >> ~/repos/qem_download_status.txt");
             my ($parent) = $maintrepo =~ 'https?://(.*)$';
             my ($domain) = $parent =~ '^([a-zA-Z.]*)';
+            record_info("test", "$parent");
```
Comment (asmorodskyi) [inline tests/publiccloud/download_repos.pm:80]: just not to forget later I will a reminder here that this needs to be removed before we will commit

**Comment by m-dati** [comment]
Please kindly add some reference in the PR header, if any poo# or motivation of this work.

**Comment by m-dati** [comment]
Description apart :smiley:, LGTM

**Review by asmorodskyi — APPROVED** [review]
LGTM

**Pull request merged**